public/style/sass/_report.scss

```diff
@@ -15,21 +9,67 @@
   z-index: 1;
 }
 
-.form-button {
-  outline:none;
+.input-label,
+.type-label,
+.type-wrap {
+  font: 15px $default;
+  font-weight: 500;
+  margin-bottom: 2px;
+}
+
+.button {
+  outline: none;
   width: 100%;
   height: 40px;
-  border: 1px solid #ccc;
-  background-color: #eee;
   font: 15px $default;
   cursor: pointer;
   padding: 0.5em;
 }
 
-.form-button:not(:last-child) {
+.form-button {
+  border: 1px solid #ccc;
+  background-color: #eee;
+}
```

Comment:
you can merge .form-button into .button - everywhere you've used one you've used the other. The only place you haven't is the .submit-button, but the only difference to .form-button there is background-color, which will override